docs/quickemu.1

```diff
@@ -37,12 +37,15 @@ You can also pass optional parameters
 Enable braille support.
 Requires SDL.
 .TP
-\f[B]\[en]delete\f[R]
-Delete the disk image.
+\f[B]\[en]delete-disk, \[en]delete\f[R]
+Delete the disk image and EFI variables
+.TP
+\f[B]\[en]delete-vm\f[R]
+Delete the entire VM and it\[cq]s configuration
 .TP
 \f[B]\[en]display\f[R]
 Select display backend.
-`sdl' (default), `gtk', `none' or `spice'
+`sdl' (default), `gtk', `none', `spice', or `spice-app'
```

Comment:
Good spot - that needs aligning in the short option section.
I'll transfer that in to the docs sub-module.